### ISSUE TITLE
Error in Updatescript when updating from 10.0.11 to 10.0.16

### ISSUE THREAD

**Issue by Abraxas84** [glpi-project/glpi#17889]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.16

### Bug description

I updated our testsystem from  GLPI 10.0.11 to 10.0.16 and when I start the database migration I get the following error:

======================================== Update auf 10.0.12 ========================================
SQL Error "1054": Unknown column 'field' in 'where clause' in query "UPDATE `glpi_tickettemplatemandatoryfields` SET `field` = '207' WHERE `field` = '205'"

I checked the update scripts and found that the script does the following:

1. **install\migrations\update_10.0.11_to_10.0.12\duplicate_searchopt_fixes.php** - This file searches the glpi_configs table for lock_use_lock_item and checks if the value is set. In our case this is TRUE.
2.  Next a list of locked items (lock_item_list) is searched and in our case this has the value: ["Ticket"]
3. Now the changeSearchOption function is called with the values "Ticket", 205, 207
4. **src\Migration.php:1653** The search options are updated with the values from the script.
5. Later in the process the  **src\Migration.php:1670** migrateSearchOptions function is running. and here we have the following part of the script starting from Line 1721:

```
if (is_a($itemtype, 'CommonITILObject', true)) {
                    $tables = [
                        'glpi_' . strtolower($itemtype) . 'templatehiddenfields',
                        'glpi_' . strtolower($itemtype) . 'templatemandatoryfields',
                        'glpi_' . strtolower($itemtype) . 'templatepredefinedfields',
                    ];
                    foreach ($tables as $table) {
                        if (!$DB->tableExists($table)) {
                            continue;
                        }
                        $DB->updateOrDie($table, [
                            'field' => $new_search_opt
                        ], [
                            'field' => $old_search_opt
                        ]);
                    }
                }
```

Since we added ticket as first parameter the following tables are searched:

1. glpi_tickettemplatehiddenfields
2. glpi_tickettemplatemandatoryfields
3. glpi_tickettemplatepredefinedfields

All three tables exist in our environment but non of them had a field column and thats why we cannot update.

**Workaround**

1. I added the field column to all 3 tables 
2. Run the migration script (no problems)
3. I removed the field columns I added earlier

Thank you for you work 👍

### Relevant log output

```shell
SQL Error "1054": Unknown column 'field' in 'where clause' in query "UPDATE `glpi_tickettemplatemandatoryfields` SET `field` = '207' WHERE `field` = '205'"
```

**Comment by cedric-anne** [comment]
Fixed by #17481.

**Comment by Abraxas84** [comment]
Ahh I see thank you 👍
